src/test/java/org/zalando/nakadi/validation/JSONSchemaValidationTest.java

```diff
@@ -37,6 +38,27 @@ public void validationOfBusinessEventShouldRequiredMetadata() {
         assertThat(error.get().getMessage(), equalTo("#: required key [metadata] not found"));
     }
 
+    @Test
+    public void validationOfBusinessEventShouldAllowSpanCtxtInMetadata() {
+        final EventType et = EventTypeTestBuilder.builder().name("some-event-type")
+                .schema(basicSchema()).compatibilityMode(CompatibilityMode.COMPATIBLE).build();
+        et.setCategory(EventCategory.BUSINESS);
+
+        final JSONObject event = new JSONObject("{\"metadata\":{" +
+                "\"occurred_at\":\"1992-08-03T10:00:00Z\"," +
+                "\"eid\":\"329ed3d2-8366-11e8-adc0-fa7ae01bbebc\"," +
+                "\"span_ctx\": {" +
+                "      \"ot-tracer-spanid\": \"b268f901d5f2b865\"," +
+                "      \"ot-tracer-traceid\": \"e9435c17dabe8238\"," +
+                "      \"ot-baggage-foo\": \"bar\"" +
```

Comment:
can we also have a test that validates the case when we have a none-string field in span_ctx? (Validation should fail in this case)

Reply:
@v-stepanov done 8488b55